src/metrics.jl

```diff
@@ -838,16 +838,16 @@ end
 #    of `_centralize` -- ~4x speed up
 _centralize_colwise(x::AbstractVector) = x .- mean(x)
 _centralize_colwise(x::AbstractMatrix) = x .- mean(x, dims=1)
-function colwise!(r::AbstractVector, dist::CorrDist, a::AbstractMatrix, b::AbstractMatrix)
+function colwise!(r::AbstractArray, ::CorrDist, a::AbstractMatrix, b::AbstractMatrix)
```

Comment:
Sorry I had missed the `AbstractArray` change. I think it should be tested, otherwise it may regress without being noticed.

Reply:
I only realized that because you corrected my newly added docstring recently. 😉 We have that signature in the generic `colwise!` methods.